EIPS/eip-FOCIL.md

```diff
@@ -61,6 +61,18 @@ This EIP introduces backward incompatible changes to the block validation rule s
 
 ## Security Considerations
 
+### Consensus Liveness
+
+The builder or proposer of slot `n+1` cannot construct a canonical block without the inclusion lists and inclusion list aggregate of slot `n`. This protocol relies on proposers being sufficiently peered with the inclusion list committee. Builders (if using MEV-boost) are sufficiently peered with the inclusion list aggregation network.
```

Comment:
The builder or proposer of slot `n+1` cannot construct a canonical block without seeing local ILs and the IL aggregate broadcast during slot `n`. This implies the block producer (e.g., a proposer or a proposer builder pair needs to be sufficiently peered with the IL committee members. The parameter Δ also needs to be set to prevent liveness issues from accidental disparities between the proposer view and attesters' views.